### PR TITLE
Cog modis up5323

### PR DESCRIPTION
**Pull request by thaisbendixen**
This PR adresses the following ticket: https://app.circleci.com/pipelines/github/up42/blocks/5043/workflows/48141538-af55-4e0e-adcd-5d3cda3e3d1e/jobs/9885.

For the Modis Block It was necessary to define the bands using ColorInterpret, as the COG conversion with rasterio seems to incorrectly assume that the last band is an alpha band.

Items:
* [ ] ...
* [x] Ran e2e and live-tests for updated blocks
* [x] Add units tests
* [x] Bumped block versions

### CODE REVIEW

File: UP42Manifest.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
       "bbox": {"type": "array", "default": null},
       "intersects": {"type": "geometry"},
       "contains": {"type": "geometry"},
-      "time": {"type": "dateRange", "default": "2018-12-01T00:00:00+00:00/2020-12-31T23:59:59+00:00"},
+      "time": {"type": "dateRange", "default": "2018-12-01T00:00:00+00:00/2021-02-15T23:59:59+00:00"},
```
Comment (rodrigoalmeida94) [inline UP42Manifest.json:18]: Date here should be 31.12.2021

File: tests/test_modis.py [hunk 2]
```diff
@@ -174,6 +175,7 @@ def test_aoiclipped_fetcher_fetch(requests_mock, modis_instance):
     assert len(result.features) == 1
 
     img_filename = "/tmp/output/%s" % result.features[0]["properties"]["up42.data_path"]
+    assert cog_validate(img_filename)[0] is True
```
Comment (rodrigoalmeida94) [inline tests/test_modis.py:178]: You can remove the `is True`

File: tests/test_modis.py [hunk 3]
```diff
@@ -477,3 +481,38 @@ def test_aoiclipped_fetcher_geom_error_fetch_live(modis_instance):
...
+    )
+
+    result = modis_instance.fetch(query, dry_run=False)
+    assert len(result.features) == 1
+    img_filename = "/tmp/output/%s" % result.features[0]["properties"]["up42.data_path"]
+    with rio.open(img_filename) as dataset:
+        band2 = dataset.read(2)
+        assert np.sum(band2) == 28202042
+        assert dataset.count == 7
+    assert cog_validate(img_filename)[0] is True
```
Comment (rodrigoalmeida94) [inline tests/test_modis.py:518]: Same here

**Review by rodrigoalmeida94 — APPROVED** [review]
Nice!

File: UP42Manifest.json (hunk 1 above)
Comment (thaisbendixen) [reply UP42Manifest.json:18]: Good catch!

File: src/gibs.py [hunk 4]
```diff
@@ -286,6 +287,14 @@ def post_process(img_filename, imagery_layers):
             img_bands_count = dst.count
             for band in make_list_layer_band(imagery_layers, img_bands_count):
                 dst.update_tags(band[0], layer=band[1], band=band[2])
+            # The COG conversion assumes last band is an alpha band therefore It's necessary to define the ColorInterp
+            # property
+            color_interp = [ColorInterp.red, ColorInterp.green, ColorInterp.blue]
```
Comment (nekhtiari) [inline src/gibs.py:292]: wow, nice, feel stupid that it didn’t come to my mind!

**Review by nekhtiari — APPROVED** [review]

File: src/gibs.py (hunk 4 above)
Comment (thaisbendixen) [reply src/gibs.py:292]: What!? I don't think It was very obvious either! Took us a while to figure out!

**Pull request merged**